substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jfr/JfrThreadLocal.java

```diff
@@ -87,6 +88,11 @@ public class JfrThreadLocal implements ThreadListener {
     private static final FastThreadLocalLong unparseableStacks = FastThreadLocalFactory.createLong("JfrThreadLocal.unparseableStacks");
     private static final FastThreadLocalWord<SamplerSampleWriterData> samplerWriterData = FastThreadLocalFactory.createWord("JfrThreadLocal.samplerWriterData");
 
+    /* ThreadCPULoad thread-locals. */
+    private static final FastThreadLocalLong cpuTime = FastThreadLocalFactory.createLong("JfrThreadLocal.cpuTime");
```

Comment:
I think it would be better to move those static fields to the class `ThreadCPULoadEvent` because they are pretty event specific (and you won't need any getters/setters).

Reply:
The `ThreadLocal` constants are moved to `ThreadCPULoadEvent` class.